src/components/Auth/types.ts

```diff
@@ -62,9 +62,19 @@ export interface AuthUser extends AddAuthUserPayload {
   providerUserInfo: AuthProviderInfo[];
 }
 
+export enum UsageModes {
+  // Passthrough mode is off
+  DEFAULT = 'DEFAULT',
+  // Passthrough mode is on
+  PASSTHROUGH = 'PASSTHROUGH',
+  // We shouldn't ever get this usage mode
+  USAGE_MODE_UNSPECIFIED = 'USAGE_MODE_UNSPECIFIED',
+}
+
 export interface AuthState {
   authUserDialogData?: RemoteResult<AuthUser | undefined>;
   users: RemoteResult<AuthUser[]>;
   filter: string;
   allowDuplicateEmails: boolean;
+  usageMode: UsageModes;
```

Comment:
Instead of an enum, we could make this a boolean instead (since `USAGE_MODE_UNSPECIFIED` shouldn't happen as far as I know). But that may make it trickier to handle more usage modes in the future, if that is a thing that is likely to happen.

Reply:
Updated to `UsageMode` and moved `USAGE_MODE_UNSPECIFIED` to the top.